packages/eslint-plugin-svelte/src/rules/require-store-callbacks-use-set-param.ts

```diff
@@ -27,7 +28,20 @@ export default createRule('require-store-callbacks-use-set-param', {
 						context.report({
 							node: fn,
 							loc: fn.loc,
-							messageId: 'unexpected'
+							messageId: 'unexpected',
+							fix: (fixer) => {
+								if (param) {
+									return fixer.replaceText(param, 'set');
+								}
```

Comment:
Renaming a parameter would break the parts of the code that use that variable. This doesn’t seem to align with ESLint’s fixer policy.

https://eslint.org/docs/latest/extend/custom-rules#applying-fixes

> Best practices for fixes:
> Avoid any fixes that could change the runtime behavior of code and cause it to stop working.
> Make fixes as small as possible. Fixes that are unnecessarily large could conflict with other fixes, and prevent them from being applied.

Reply:
You're right. It does certainly align with the best practices, I just overlooked this part. We should be updating references to the parameter with the new name